go-sdk/pkg/logging/shclog/shclog.go

```diff
@@ -31,10 +31,9 @@ import (
 	"time"
 
 	"github.com/hashicorp/go-hclog"
-)
 
-// Slog does not have built in trace log level
-const SlogLevelTrace = slog.LevelDebug - (slog.LevelInfo - slog.LevelDebug)
```

Comment:
We use `SlogLevelTrace` at other places in the code, don't we need it?

Reply:
Moved elsewhere (I think I forgot to to push it in this commit)